plugins/sqlite/sqlitedatabase.cpp

```diff
@@ -37,21 +37,21 @@ Q_DECLARE_OPAQUE_POINTER(sqlite3*)
 Q_DECLARE_METATYPE(sqlite3*)
 
 // custom sqlite function "comparePhoneNumbers" used to compare IDs if necessary
-void comparePhoneNumbers(sqlite3_context *context, int argc, sqlite3_value **argv)
+void comparePhoneNumbers(sqlite3_context *context, int /* argc */, sqlite3_value **argv)
 {
     QString arg1((const char*)sqlite3_value_text(argv[0]));
     QString arg2((const char*)sqlite3_value_text(argv[1]));
     sqlite3_result_int(context, (int)History::PhoneUtils::comparePhoneNumbers(arg1, arg2));
 }
 
-void compareNormalizedPhoneNumbers(sqlite3_context *context, int argc, sqlite3_value **argv)
+void compareNormalizedPhoneNumbers(sqlite3_context *context, int /* argc */, sqlite3_value **argv)
 {
     QString arg1((const char*)sqlite3_value_text(argv[0]));
     QString arg2((const char*)sqlite3_value_text(argv[1]));
     sqlite3_result_int(context, (int)History::PhoneUtils::compareNormalizedPhoneNumbers(arg1, arg2));
 }
 
-void normalizeId(sqlite3_context *context, int argc, sqlite3_value **argv)
+void normalizeId(sqlite3_context *context, int /* argc */, sqlite3_value **argv)
 {
     QString accountId((const char*)sqlite3_value_text(argv[0]));
```

Comment:
Might be good to add a check on `argc` being >= 2

Reply:
I agree, let me fix that

Reply:
That should do :)